lib/stigtest.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Base module for STIG test cases
+# Maintainer: llzhao <llzhao@suse.com>
+
+package stigtest;
+
+use strict;
+use warnings;
+use testapi;
+use utils;
+use base 'opensusebasetest';
+
+our @EXPORT = qw(
+  $profile_ID
+  $f_ssg_sle_ds
+  $f_stdout
+  $f_stderr
+  $f_report
+  $remediated
+  set_ds_file
+  upload_logs_reports
+);
+
+# The file names of scap logs and reports
+our $f_stdout = 'stdout';
+our $f_stderr = 'stderr';
+our $f_report = 'report.html';
+
+# Set default value for 'scap-security-guide' ds file
+our $f_ssg_sle_ds = '/usr/share/xml/scap/ssg/content/ssg-sle12-ds.xml';
```

Comment:
@lilyeyes Would you provide the VR for SLE12 as you have the code about ssg-sle12-ds.xml ?

Reply:
I have discussed with Marcus I will only support sle15. IMO qem team might be like to add it to their job group.

Reply:
got it :)